wled00/FX_fcn.cpp

```diff
@@ -1566,12 +1569,19 @@ void WS2812FX::deserializeMap(uint8_t n) {
 
   JsonArray map = doc[F("map")];
   if (!map.isNull() && map.size()) {  // not an empty map
-    customMappingSize  = map.size();
+    // if isMatrix then customMap size is whole matrix
```

Comment:
Considering other comments it may be better to move the logic of mapping "logical to physical" into this area and leave `setUpMatrix()` as it was (allowing #2892)

Reply:
Okay, wait for https://github.com/Aircoookie/WLED/pull/2892 ...

I don't see ledmaps and 2D logical to physical working without adapting setUpMatrix as setUpMatrix must be aware of both the ledmap and the 2D mapping and apply both of them.